lib/recog/nizer.rb

```diff
@@ -41,6 +41,24 @@ def self.match(match_key, match_string)
     nil
   end
 
+  def self.multi_match(match_key, match_string)
```

Comment:
That is OK -- Metasploit pins its version of recog at a specific version, so even if we were to release this in a minor version, they wouldn't get it until they explicitly bumped.  But the right way to do this would be to do the API-breaking change in a major version bump.  

For now, though, yes, lets keep it as is.  This gets us what we need short term, doesn't break metasploit, allows us to deal with the problems that multi match will likely uncover and conveniently avoids the problem of how metasploit would handle multiple matches (can be done, but requires work on that end).